packages/debug/src/common/debug-preferences.ts

```diff
@@ -64,7 +64,18 @@ export const debugPreferencesSchema: PreferenceSchema = {
             description: nls.localizeByDefault('Show Source Code in Disassembly View.'),
             type: 'boolean',
             default: true,
-        }
+        },
+        'debug.autoExpandLazyVariables': {
+            type: 'string',
+            enum: ['auto', 'on', 'off'],
+            default: 'auto',
+            enumDescriptions: [
+                nls.localizeByDefault('When in screen reader optimized mode, automatically expand lazy variables.'),
+                nls.localizeByDefault('Always automatically expand lazy variables.'),
+                nls.localizeByDefault('Never automatically expand lazy variables.')
+            ],
+            description: nls.localizeByDefault('Controls whether variables that are lazily resolved, such as getters, are automatically resolved and expanded by the debugger.')
+        },
```

Comment:
As far as i understand it the auto option is currently not implemented, right? I would then prefer to not have it available at all and remove it for now.